configs/read-only-hub.properties

```diff
@@ -0,0 +1,35 @@
+app.name=hub-v2
+app.environment=staging
+app.lib_path=/opt/hub/lib/
+app.url=http://localhost/
+app.maxPayloadSizeMB=40
+
+hub.runGC=true
+hub.gcMinutes=5
+hub.protect.channels=true
+hub.webhooks.enable=false
+
+# spoke
+spoke.write.path=/mnt/spoke/write
+spoke.read.path=/mnt/spoke/read
+spoke.write.ttlMinutes=60
+spoke.read.ttlMinutes=10
+
+# aws
+aws.signing_region=us-east-1
+aws.credentials=/etc/hub/credentials.properties
```

Comment:
Dumb question, but does this need to get updated to `read-only-credentials.properties`? 

Also, I'm guessing the AWS credentials files aren't here because they're being managed by salt right now and docker stuff doesn't run in that mode... if we run locally in AWS mode, we'd need to put one somewhere manually, too?

Reply:
a) It's a little confusing but the original file name, `read-only-credentials.properties`, gets renamed to `credentials.properties` when it's put into the Docker image.

b) You are correct. I can see the read-only credentials being in salt also but since I was still hacking around I had them in the repo (with fake info) that I have a modified version of locally that actually works.